contributors/devel/sig-architecture/platforms.md

```diff
@@ -0,0 +1,129 @@
+# Shipping alternate platforms in Kubernetes release artifacts
+
+The default Kubernetes platform is Linux/amd64. This platform is fully tested
+and build and release systems initially supported only that. A while ago we
+started an [effort to support multiple architectures][0]. As part of this
+effort, we added support in our build/release pipelines for the architectures
+arm, arm64, ppc64le and s390x on different operating systems like Linux, Windows
+and macOS.
+
+[0]: https://github.com/kubernetes/kubernetes/issues/38067
+
+The main focus was to have binaries and container images to be available for
+these architectures/operating systems and for contributors that are interested
+to be able to take these artifacts and set up CI jobs to adequately test these
+platforms. Specifically to call out the ability to run conformance tests on
+these platforms.
+
+Target of this document is to provide a starting point for adding new platforms
+to Kubernetes from a SIG Architecture perspective. This does not include release
+mechanics or supportability in terms of functionality.
+
+# Step 1: crawling (Build)
+
+- bazel based build infrastructure should support this architecture
```

Comment:
`bazel based ...` this is going to be funny :) ;)

Reply:
I second that! bazel on non-amd64 architectures will always be challenging.

Reply:
Right now we build the binary artifacts via `make cross-in-a-container` which uses the kube-cross container image, which does not seem to use bazel.

Reply:
that's just the release process. regular folks should be still able to use bazel like they do for development workflows. (UNTIL we get of bazel entirely)

Reply:
Agreed :+1:

Reply:
Not only does the Bazel binary need to support any new architecture, but also the various and sundry rules files need to work as well. (Currently tracking a specific rules_docker dependency for arm64, for example, which is blocking `kind`.) See https://github.com/kubernetes-sigs/kind/issues/166

Reply:
i do not see why bazel is a strong requirement.
is the language 'should' implying MUST or MAY?

Reply:
Bazel on s390x is quite problematic too. Looks like it applies to other arch too.

Reply:
Removed the bazel restriction :heavy_check_mark: